mitodl/open-discussions
# cover image and post creation form tweaks

#### Pre-Flight checklist


#### What are the relevant tickets?

part of #1706 
closes #1704

#### What's this PR do?

some tweaks and changes to the post creation page and to the article cover image UI in general.

- contents of the dropzone change (see #1706 for what it should be)
- added a 'remove cover image' button on the `CoverImageInput`, which causes the cover image input to be hidden.
- made the background and surroundings for the form white, so the form somewhat resembles the way that the post will actually be displayed after you create it.

#### How should this be manually tested?

read through the issues and see the tweaks that should be made. ensure they're all there on the create post page and when editing an existing article post, and ensure that no existing functionality is negatively impacted.


#### Screenshots (if appropriate)
![emptyformfasdfasdfpoastpagfff](https://user-images.githubusercontent.com/6207644/51276761-a596d500-19a3-11e9-9676-6a3b1f081272.png)
![emptyformfasdfasdfpoastpag](https://user-images.githubusercontent.com/6207644/51276762-a596d500-19a3-11e9-87d4-8f51dc6bef8a.png)
![emptyformfasdfasdf](https://user-images.githubusercontent.com/6207644/51276763-a62f6b80-19a3-11e9-900a-21cd3b0435e6.png)
![emptyformfasdf](https://user-images.githubusercontent.com/6207644/51276764-a62f6b80-19a3-11e9-8fcd-3c905327ed06.png)
